src/index.tsx

```diff
@@ -117,9 +119,10 @@ const getContainerElementById = (containerId: string) => {
   )
 }
 
-export const init = (opts: SdkOptions): SdkHandle => {
+export const init = (opts: SdkOptions, conf?: SdkConfiguration): SdkHandle => {
```

Comment:
This would allow anyone to override the configurations. I think we should only allow this if the environment is `test` (so for surge demo app and for E2E tests)